voila/utils.py

```diff
@@ -94,16 +103,86 @@ def get_page_config(base_url, settings, log):
     page_config.setdefault("fullMathjaxUrl", mathjax_url)
 
     labextensions_path = jupyter_path("labextensions")
+
     recursive_update(
         page_config,
         gpc(
             labextensions_path,
             logger=log,
         ),
     )
+    disabled_extensions = [
+        "@voila-dashboards/jupyterlab-preview",
+        "@jupyter/collaboration-extension",
+    ]
+    must_have_extensions = ["@jupyter-widgets/jupyterlab-manager"]
```

Comment:
Maybe `required_extensions` would also work as an alternative to `must_have_extensions`?